src/ui/layout/RecentObjectsList.vue

```diff
@@ -179,6 +215,7 @@ export default {
       while (this.recents.length > MAX_RECENT_ITEMS) {
         const poppedRecentItem = this.recents.pop();
         this.removeCompositionListenerFor(poppedRecentItem.navigationPath);
+        this.removeNameListenerFor(poppedRecentItem.domainObject);
```

Comment:
do we also need to remove listeners for each path object in the items breadcrumb?

Reply:
Ah, I wasn't deleting the actual listeners, so added that [here](https://github.com/nasa/openmct/pull/6927/files#diff-f7e43a55287ea63b0298fad0bd268b18cc6fc5b62d39578cbb9df112eaacd3cdR100). Is that what you mean?

Reply:
On unmounted, we're [removing all the listeners](https://github.com/nasa/openmct/pull/6927/files#diff-f15ad3f53c216ea726d2c832828cf25c1573f59bd708610eab3f8db8c15aedc0R80)